test/e2e/deploy/deploy.go

```diff
@@ -622,16 +642,45 @@ func EnsureDeployment(deploymentName string) *Deployment {
 				if deployment.HasOperator {
 					WaitForOperator(c, deployment.Namespace)
 				}
+				if deployment.HasOLM {
+					WaitForOLM(c, "olm")
+				}
 				return
 			}
 			framework.Logf("have %s PMEM-CSI deployment, want %s -> delete existing deployment", running.Name, deployment.Name)
+
+			if running.HasOLM {
+				cmd := exec.Command("test/stop-operator.sh", "-olm")
+				cmd.Dir = os.Getenv("REPO_ROOT")
+				cmd.Env = append(os.Environ(),
+					"TEST_OPERATOR_NAMESPACE="+running.Namespace,
+					"TEST_OPERATOR_DEPLOYMENT="+running.Name)
+				cmd.Stdout = ginkgo.GinkgoWriter
+				cmd.Stderr = ginkgo.GinkgoWriter
+				err = cmd.Run()
```

Comment:
Can you use https://github.com/intel/pmem-csi/commit/40ba3555a43fa2aa8aa1f081af1f2386edda4dd8 here?

Reply:
Ok, Done.